assets/js/utils/canvasUtils.ts

```diff
@@ -0,0 +1,84 @@
+import { FRAME_RATE, SCREEN_CONSTRAINTS, CANVAS_HEIGHT, CANVAS_WIDTH } from "./const";
+import type { MergedScreenRef } from "./rtcUtils";
+
+export const getMergedTracks = async (
+  mergedScreenRef: MergedScreenRef,
+  presenterStream: MediaStream
+) => {
+  if (mergedScreenRef.refreshId) stopRefreshingAndRemovePreviousScreen(mergedScreenRef);
```

Comment:
Could we make it like this?
```suggestion
  mergedScreenRef.refreshId && stopRefreshingAndRemovePreviousScreen(mergedScreenRef);
```